Tests/scripts/utils/tests/content_packs_util_test.py

```diff
@@ -65,4 +68,14 @@ def test_should_test_content_pack(pack_name, expected):
         - Case A: Verify pack should be tested
         - Case B: Verify pack should not be tested
     """
+    # Creating temp dirs
+    packs_dir = tmp_path / PACKS_DIR
+    packs_dir.mkdir()
+    pack = packs_dir / pack_name
+    pack.mkdir()
```

Comment:
no need to create the directories in 2 seperate calls, you can do the following:
```python
pack = tmp_path / PACKS_DIR / pack_name
pack.mkdir(parents=True)
```

Reply:
Where did you find this? Didn't manage to find anything like this in docs.